content/applications/sales/sales/ebay_connector/setup.rst

```diff
@@ -1,126 +1,172 @@
-==============================
-How to configure eBay in Odoo?
-==============================
+==================
+eBay configuration
+==================
 
 Create eBay tokens
 ==================
 
-In order to create your tokens, you need to create a developer account on the
-`developer portal <https://go.developer.ebay.com/>`_.
-Once you are logged in, you can create **Sandbox Keys** and **Production Keys**
-by clicking on the adequate buttons.
+First, create a *Developer* account with eBay via the `developer portal
+<https://go.developer.ebay.com/>`_.
+
+.. note::
+   It will take one business day for eBay to confirm the status of the account.
```

Comment:
I would consider changing this up just to give us some plausible deniability. "May take up to one business day" or something similar. (Unless you took this directly from eBay, then feel free to ignore)